app/services/users.js

```diff
@@ -31,4 +31,20 @@ const getUsers = async ({ limit = 10, offset = 0 }) => {
   }
 };
 
-module.exports = { createUser, access, getUsers };
+const createAdmin = async body => {
+  const { email } = body;
+  try {
+    const user = await db.User.findOne({ where: { email } });
+    if (!user) {
+      const newUser = await db.User.newUser({ ...body, role: 'admin' });
+      return Promise.resolve(newUser);
```

Comment:
Remember this `Promise.any()` functions fould not be use in async/await functions